[6.20] Fix CMake 'pyroot' option renaming (#5132)
root-project/root: README/ReleaseNotes/v620/index.md

## Build, Configuration and Testing Infrastructure

> - Option `python` was renamed to `pyroot`. While still using `python` option, `pyroot` will be enabled authomatically.

Reviewer: I think the following captures the "automatic" part and mentions the deprecation:

```suggestion
- Option `python` was renamed to `pyroot`. The option spelled `python` is a deprecated alias for `pyroot`.
```
